01-Login/Auth0Sample/HybridAuth.swift

```diff
@@ -167,6 +167,20 @@ import Auth0
                 }
         }
     }
+    
+    @objc
+    func logOutUser(callback: @escaping(Bool, String) -> Void){
+        Auth0
+            .webAuth()
+            .clearSession(federated: true){
+                switch $0 {
+                case true:
+                    callback($0, "Logged out successfully")
```

Comment:
I would not have this method decide on the language to be used for the message. I would return the outcome bool back to the ViewController and make that decision there. You can also drop all of the `switch` code and just have `callback($0)`